.travis.yml

```diff
@@ -3,7 +3,7 @@ language: node_js
 node_js:
```

Comment:
@bwbuchanan maybe just delete this file and provide GitHub actions config instead?